reference/docs-conceptual/components/vscode/How-To-Replicate-the-ISE-Experience-In-VSCode.md

```diff
@@ -27,12 +27,29 @@ to make the user experience a bit more familiar compared to the ISE.
 You can [configure your own key bindings](https://code.visualstudio.com/docs/getstarted/keybindings#_custom-keybindings-for-refactorings)
 in VSCode as well.
 
+## Simplified ISE-like UI
+
+If you're looking to simplify the Visual Studio Code UI to look more closely to the UI of the ISE, apply these two settings:
+
+```json
+"workbench.activityBar.visible": false,
+"debug.openDebug": "neverOpen",
+```
+
+This will hide the "Activity Bar" and the "Debug Side Bar" sections below inside of the red box:
+
+![highlighted section includes Activity Bar and Debug Side Bar](https://user-images.githubusercontent.com/2644648/68706263-52020300-0544-11ea-94fa-0bc4b5a7f4e1.png)
```

Comment:
Please create the folder `reference\docs-conceptual\components\vscode\images\How-To-Replicate-the-ISE-Experience-In-VSCode` and copy your images. Change the filenames to something short but descriptive. See `Using-VSCode-for-Remote-Editing-and-Debugging.md` for an example of how to include images.